### PR TITLE
Bug fix and version bump

### PR DESCRIPTION
**Pull request by FelixMoelder**
This PR fixes an issue, identified by @christopher-schroeder, where vcf-content is written into a bcf-file without transforming into binary output.

Additionally, there are some version updates circumventing potential issues in the workflow.

### CODE REVIEW

**Comment by johanneskoester** [comment]
There is a conda version conflict in one of the envs.

**Comment by FelixMoelder** [comment]
It seems we have a little issue here.


> There is a conda version conflict in one of the envs.

It seems like we need to go back to bcftools 1.9 for igv-reports as pysam doesn't support v1.10, yet.
But as igv-reports at the end of line that should not be any problem.

**Comment by FelixMoelder** [comment]
> Thanks! Please revert the config file changes, I'd like to keep the homo sapiens example for now.

Oooops. That was unintended.

File: workflow/rules/annotation/pass2.smk [hunk 1]
```diff
@@ -44,7 +44,7 @@ rule annotate_dgidb:
     resources:
         dgidb_requests=1
     shell:
-        "rbt vcf-annotate-dgidb {input} > {output}"
+        "rbt vcf-annotate-dgidb {input} | bcftools view -Ob > {output}"
```
Comment (johanneskoester) [inline workflow/rules/annotation/pass2.smk]: Sorry, just seen this. I'd rather have rbt print BCF directly. This is how all other rbt commands should be implemented already. Can you change that accordingly?

Comment (FelixMoelder) [reply workflow/rules/annotation/pass2.smk]: Sure, I will check on that.

**Comment by FelixMoelder** [comment]
The last commit will work as soon as rbt is updated to a new version (https://github.com/rust-bio/rust-bio-tools/pull/46)

**Pull request merged**